Source/WebScheduler.FrontEnd.BlazorApp/Pages/ScheduledTask/Index.razor

```diff
@@ -108,6 +109,10 @@
                 this.totalScheduledTasks = ret.TotalCount;
                 scheduledTasks = ret.Items;
             }
+            catch (AccessTokenNotAvailableException exception)
```

Comment:
This is now required because of the breaking changes introduced in 7.0. token comes from history state for better security.

Also gets a token if one hasn't been exchanged yet.